orca-core/src/main/java/com/netflix/spinnaker/orca/pipeline/model/StageContext.java

```diff
@@ -64,6 +64,16 @@ public Object get(@Nullable Object key) {
     }
   }
 
+  /**
+   * Get a value from the current context ONLY - never looking at the ancestors' outputs
+   *
+   * @param key The key to look
+   * @param defaultValue default value to return if key is not present
+   * @return value or null if not present
+   */
+  Object getCurrentOnly(@Nullable Object key, Object defaultValue) {
```

Comment:
Not sure how I feel about `getCurrentOnly` vs overloading `get()` -- will let someone else weigh in.

Reply:
you mean something like 
```
public Object get(@Nullable Object key, Object defaultValue, boolean localOnly)
```
?

Reply:
+1 on overloading `get(...)`

Reply:
Just curious, why?

I think
```
stage.context.getCurrentOnly("someValue", false);
```

is more descriptive than:
```
stage.context.get("someValue", false, true);
```

Reply:
get() because the context is a map. Matter of preference.

Reply:
I hear you. This area has caused some confusion in the past (where something comes from in the context) so I am more inclined to make this very explicit